internal/sdkv2provider/resource_cloudflare_record.go

```diff
@@ -501,3 +502,22 @@ func suppressTrailingDots(k, old, new string, d *schema.ResourceData) bool {
 
 	return strings.TrimSuffix(old, ".") == newTrimmed
 }
+
+func suppressMatchingIpv6(_, old, new string, _ *schema.ResourceData) bool {
```

Comment:
if you're not using these extra parameters, just get rid of them.

```suggestion
func suppressMatchingIpv6(old, new string) bool {
```